apps/www/registry/sydney/example/alert-animation.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react"
+// Import framer-motion
+import {
+  CheckCircledIcon,
+  CrossCircledIcon,
+  ExclamationTriangleIcon,
+  InfoCircledIcon,
+  RocketIcon,
+} from "@radix-ui/react-icons"
+import { motion } from "framer-motion"
+import { AlertCircle, Terminal } from "lucide-react"
+
+import { Alert, AlertDescription, AlertTitle } from "@/registry/sydney/ui/alert"
+
+// 定义动画变体对象
```

Comment:
Please do not attach non-English comments.

Reply:
i will check my code right now

Reply:
Ive deleted these comments.Thank you !